pallets/fragments/src/tests.rs

```diff
@@ -2100,6 +2100,11 @@ mod create_account_tests {
 	}
 }
 
+mod resell_tests {
```

Comment:
@alebaffa @sinkingsugar, what is our "testing philosophy" 😅 . Do we only want to test "public functions"?

Because if you notice that `fragments.resell()` uses the same private function `fragments.can_transfer_instance()` as `fragments.give()` - so a lot of the test cases will be similar. for example: `give_tests.give_should_work_if_the_expiration_parameter_is_greater_than_the_current_block_number()`, `give_test.give_should_not_work_if_the_new_permissions_are_less_restrictive()`

So I don't wanna waste time writing the exact same tests in `resell_tests`. But I'm not sure what's the correct convention/approach 😅?

Reply:
Everything needs to be tested. Of course no need to replicate tests. If a private function is already tested once, that's enough.